commons/feature_toggles/feature_toggle_manager.py

```diff
@@ -0,0 +1,69 @@
+import os
+import json
+from time_tracker_api.security import current_user_email
+from azure.appconfiguration import AzureAppConfigurationClient
+
+
+class MSConfig:
+    def check_variables_are_defined():
+        auth_variables = ['AZURE_APP_CONFIGURATION_CONNECTION_STRING']
+        for var in auth_variables:
+            if var not in os.environ:
+                raise EnvironmentError(
+                    "{} is not defined in the environment".format(var)
+                )
+
+    check_variables_are_defined()
+    AZURE_APP_CONFIGURATION_CONNECTION_STRING = os.environ.get(
+        'AZURE_APP_CONFIGURATION_CONNECTION_STRING'
+    )
+
+
+class FeatureToggleManager:
+    def __init__(self, key: str, label: str = None, config=MSConfig):
+        self.key = key
+        self.label = label
+        self.config = config
+        self.client = self.get_azure_app_configuration_client()
+        self.configuration = {}
+
+    def get_azure_app_configuration_client(self):
+        connection_str = self.config.AZURE_APP_CONFIGURATION_CONNECTION_STRING
+        client = AzureAppConfigurationClient.from_connection_string(
+            connection_str
+        )
+
+        return client
+
+    def get_configuration(self, key: str, label: str):
+        configuration = self.client.get_configuration_setting(
+            key=f".appconfig.featureflag/{self.key}", label=self.label
```

Comment:
We are using `self.key` and `self.label` variables in the function. But we are also passing key, and label as arguments in the function call.

We could use `self.var` or the `var` . I would suggest use argument vars.

Reply:
We already change to use the argument `vars` in the function.